boards/arm/nrf5340_dk_nrf5340/Kconfig

```diff
@@ -0,0 +1,18 @@
+# Kconfig - nRF5340 DK NRF5340 board configuration
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
```
# Kconfig - nRF5340 DK NRF5340 board configuration
# Copyright (c) 2019 Nordic Semiconductor ASA
(blank line here)
# SPDX-License-Identifier: Apache-2.0
```

(Nit from old PR.)

Reply:
Ah right, so you want a blank line completely, not with `#` :)

Reply:
OK @ulfalizer hold on :)

Reply:
Skip the `Kconfig - ...` part too. It's clear already that it's about Kconfig.

Won't keep nitting these for all PRs. It's mostly to indirectly make people aware that there's some consistency now.

Reply:
@ulfalizer done now :)

Reply:
Oh, wait, a sec, this may be missing.

Reply:
Note that there's also a bunch of other similar headers to be fixed (e.g. in boards/arm/nrf5340_dk_nrf5340/Kconfig.board).

Reply:
Fixed now, thanks :)